### PR TITLE
dotCMS/core#13026 /edit does not deliver the logged in user to the last page browsed on front end

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts [hunk 1]
```diff
@@ -67,9 +67,15 @@ export class IframePortletLegacyComponent implements OnInit {
      * Tigger when the current site is changed, this method reload the iframe if is neccesary
      * @memberof IframePortletLegacyComponent
      */
-    reloadIframePortlet(): void {
+    reloadIframePortlet(portletId?: string): void {
         this.dotLoadingIndicatorService.show();
-        this.setUrl(this.url.getValue());
+             if (portletId) {
```
Comment (hmoreras) [inline src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts]: Indentation here

Comment (fmontes) [reply src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts]: Fixed

File: src/app/view/components/dot-navigation/dot-navigation.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+    it('should reload current portlet', () => {
+        spyOn(dotRouterService, 'reloadCurrentPortlet');
+        dotNavigationService.reloadCurrentPortlet('hello');
+
+        expect(dotRouterService.reloadCurrentPortlet).toHaveBeenCalledWith('hello');
+    });
+
+    it('should NOT reload current portlet', () => {
+        spyOn(dotRouterService, 'reloadCurrentPortlet');
+        dotNavigationService.reloadCurrentPortlet('123');
```
Comment (freddyucv) [inline src/app/view/components/dot-navigation/dot-navigation.service.spec.ts]: why do the reloadCurretPortlet method have a 'id' parameter?

File: src/app/view/components/dot-navigation/dot-navigation.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+        expect(dotNavigationService.goToFirstPortlet).not.toHaveBeenCalled();
+    });
+
+    it('should reload navigation, set menu and NOT redirect to first portlet (user redirect)', () => {
+        spyOn(dotMenuService, 'reloadMenu').and.returnValue(Observable.of(mockMenu));
+        spyOn(dotMenuService, 'isPortletInMenu').and.returnValue(Observable.of(true));
+        spyOn(dotNavigationService, 'goToFirstPortlet').and.returnValue(Observable.of(false));
+
+        loginService.triggerNewAuth(baseMockAuth);
+        expect(dotNavigationService.goToFirstPortlet).not.toHaveBeenCalled();
```
Comment (freddyucv) [inline src/app/view/components/dot-navigation/dot-navigation.service.spec.ts]: This test is the same as the previous one, I think you missed

File: src/app/view/components/dot-navigation/dot-navigation.service.spec.ts (hunk 2 above)
Comment (fmontes) [reply src/app/view/components/dot-navigation/dot-navigation.service.spec.ts]: Because we compare if the url have the same "id" the it reloads.

**Pull request merged**